build/root/WORKSPACE

```diff
@@ -2,8 +2,8 @@ load("//build:workspace_mirror.bzl", "mirror")
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "4b2c61795ac2eefcb28f3eb8e1cb2d8fb3c2eafa0f6712473bc5f93728f38758",
-    urls = mirror("https://github.com/bazelbuild/rules_go/releases/download/0.10.2/rules_go-0.10.2.tar.gz"),
+    sha256 = "1868ff68d6079e31b2f09b828b58d62e57ca8e9636edff699247c9108518570b",
+    urls = mirror("https://github.com/bazelbuild/rules_go/releases/download/0.11.1/rules_go-0.11.1.tar.gz"),
```

Comment:
we should bump this instead to 0.10.4 (which still has the necessary go 1.10.2 support); 0.11.x is going to require BUILD updates, which will be more invasive.

Reply:
Changed to 0.10.4, let's do the 0.11.x update in another PR.